WordPressVIPMinimum/Sniffs/Filters/RestrictedFilterSniff.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * WordPressVIPMinimum Coding Standard.
+ *
+ * @package VIPCS\WordPressVIPMinimum
+ */
+
+namespace WordPressVIPMinimum\Sniffs\Filters;
+
+use PHP_CodeSniffer_File as File;
+use PHP_CodeSniffer_Tokens as Tokens;
+
+/**
+ * This sniff restricts usage of some filters
+ *
```

Comment:
Add a `@since` version number for whatever the next version will be (presumably `0.4.0`).